package.json

```diff
@@ -5,24 +5,28 @@
   "main": "dist/index.js",
   "scripts": {
     "clean": "rimraf dist",
-    "transpile": "tsc",
+    "build": "npm run clean && npm run make-dist && npm run transpile",
     "make-dist": "mkdirp dist",
-    "proto": "pbjs --target static-module --path substrait-spec/proto -o dist/index.js substrait/plan.proto",
-    "transpileProto":"pbts -o dist/definitions.d.ts dist/index.js",
-    "prepare": "npm run clean && npm run make-dist && npm run proto && npm run transpileProto && npm run transpile",
+    "clean-proto": "rimraf src/generated && mkdirp src/generated",
+    "proto": "pbjs --target static-module --path substrait-spec/proto -o src/generated/definitions.js substrait/plan.proto",
+    "transpile": "tsc",
+    "transpileProto": "pbts -o src/generated/definitions.d.ts src/generated/definitions.js",
```

Comment:
The .d.ts file needs to match the .js file for the defintions file to be properly located.  Also, now that we have parser.ts it doesn't make sense for the protobuf defintiions to be the index.